Author: Unknown
Description: Correct library name from agg to agg_pic for link

diff -urNad agg-2.3~/libagg.pc.in agg-2.3/libagg.pc.in
--- agg-2.3~/libagg.pc.in	2005-02-18 03:27:48.000000000 +0100
+++ agg-2.3/libagg.pc.in	2006-03-01 18:08:26.906950512 +0100
@@ -6,5 +6,5 @@
 Name: libagg
 Description: Anti Grain Geometry - A High Quality Rendering Engine for C++
 Version: @VERSION@
-Libs: -L${libdir} -Wl,-rpath,${exec_prefix}/lib -lagg
+Libs: -L${libdir} -Wl,-rpath,${exec_prefix}/lib -lagg_pic
 Cflags: -I${includedir}
